container.go

```diff
@@ -67,6 +67,7 @@ type ContainerRequest struct {
 	Cmd            []string
 	Labels         map[string]string
 	BindMounts     map[string]string
+	VolumeMounts     map[string]string
```

Comment:
Source Format (SF) Your editor should update formatting on save, could you double check it does? 😉

Reply:
Correct. I forgot to format it.